### PR TITLE
feat(doc): add waiting queue documentation

### PR DESCRIPTION
**Pull request by ltorje-8x8**
Documentation on how to implement a waiting queue backend and examples on how to use it.

### CODE REVIEW

**Comment by damencho** [comment]
Maybe create a sub folder in extra large conferences, name waiting-queue and move all images and files there.

File: resources/waiting-queue/waiting-queue.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+This endpoint should accept only moderator's jaas tokens for a conference specified as param to the endpoint and the token to be valid for that room. The token for moderator contains:
+```
+context: {
+    user: {
+        moderator: true
+    }
+}
+```
+It allows visitors to connect to the /moderator websocket and wait for the status message to be published on /secured/conference/state/topic.{conference} topic (triggered every 15 seconds).
```
Comment (bgrozev) [inline resources/waiting-queue/waiting-queue.md:30]:
```suggestion
It allows moderators to connect to the /moderator websocket and wait for the status message to be published on /secured/conference/state/topic.{conference} topic (triggered every 15 seconds).
```

File: resources/waiting-queue/waiting-queue.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# Waiting queue
+
+Visitors queue service is used for managing the visitors queue for the 8x8 video meetings by keeping visitors websocket connections opened and when a moderator opens the meeting, the visitors are notified and allowed to join the meeting.
+The moderators should be able to see the visitors count.
+
+## Authentication
```
Comment (bgrozev) [inline resources/waiting-queue/waiting-queue.md:6]: Mention how the token is transported? I.e as a header in the STOMP connect frame (if I understand correctly)

**Comment by saghul** [comment]
I'd say this PR should really go into the handbook, we do have a section on cloud API: https://jitsi.github.io/handbook/docs/devops-guide/cloud-api/

As a further improvement, we could consider using this plugin to make it interactive: https://github.com/PaloAltoNetworks/docusaurus-openapi-docs

**Review by damencho — APPROVED** [review]

**Comment by saghul** [comment]
Why are we landing this here? It should really be in the handbook.

**Pull request merged**